modules/core/src/v2/pendingApproval.ts

```diff
@@ -303,6 +318,37 @@ export class PendingApproval {
       .asCallback(callback);
   }
 
+  getCryptographicApprovalPayload(pendingApproval: PendingApprovalData, callback?: NodeCallback<any>): Bluebird<any> {
+    const self = this;
+    return co(function*() {
+      // this function is currently overkill for a getter, but we should in
```

Comment:
Good thinking, but you could simplify this to
```
return Bluebird.resolve(pendingApproval.signingPolicy.payload).asCallback(callback);
```
and get rid of the rest of the boilerplate until it's needed.